packages/web-ui/src/ds/molecules/Chat/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './Message'
+export * from './ErrorMessage'
+export * from './MessageList'
```

Comment:
Added a few chat-related molecules to DRY. These are used both in the `chat` and `preview` modes of the Playground.